docs/callout-card.md

```diff
@@ -2,37 +2,37 @@
 
 ## Callout card
 
-`polaris-callout-card` implements the [Polaris Callout card component](https://polaris.shopify.com/components/structure/callout-card).
+`PolarisCalloutCard` implements the [Polaris Callout card component](https://polaris.shopify.com/components/structure/callout-card).
 
 ### Examples
 
 Inline usage without secondary action:
 
 ```hbs
-{{polaris-callout-card
-  title="New feature"
-  text="This new feature is awesome!"
-  primaryAction=(hash
+<PolarisCalloutCard
+  @title="New feature"
+  @text="This new feature is awesome!"
+  @primaryAction=(hash
     text="Take a look"
-    onAction=(action "showNewFeature")
+    onAction=(fn this.handleAction)
```

Comment:
Any reason for changing this name? It doesn't associate so clearly with the example card content like this 🤔

Reply:
haha, no
will restore